README.md

```diff
@@ -56,3 +58,7 @@ cargo test
 ## Minimum requirements
 
 - GCC 4.9 or higher is required.
```

Comment:
Since you're here, can you update this line to say (`GCC 4.9 or higher is required (if parsing coverage artifacts generated by GCC)`).